app/components/widgets/AlertBox.vue

```diff
@@ -87,12 +87,12 @@
             </div>
             <div class="variation-tile__toolbar">
               <i
-                v-if="variation.id !== 'default'"
+                v-if="variation.id.indexOf('default')"
```

Comment:
Generally the way people use `indexOf`` just to check for presence of the string is `foo.indexOf('bar') > -1`

Reply:
weird, I tested and it successfully removed the trash can and pencil icons from default variations, but custom variations still had them. will do some more investigating.

Reply:
@andycreeth if I change it to `variation.id.indexOf('default') > -1`, it has the opposite of my intended effect. the icons are removed from custom variations but stay on the default variation

Reply:
although based on your description of how indexOf works I'm not sure why 🤔

Reply:
got it. I don't _think_ the result will ever be anything other than -1 or 0, but I suppose it's good to be more precise just in case that can/does happen

Reply:
Nevermind I get it now.  You *only* want this to evaluate to false if the string starts with `default`.  All other cases are true.  Makes sense.